### PR TITLE
feat: add chain, balance to account info

### PR DESCRIPTION
**Pull request by eragon512**
**What changed? Why?**
- add chain, balance to account info

**Notes to reviewers**
Related OnchainKit PR: https://github.com/coinbase/onchainkit/pull/155

**How has it been tested?**
Manual

Screenshot 2024-02-20 at 9 37 29 PM

### CODE REVIEW

File: template/web/src/components/header/AccountInfoPanel.tsx [hunk 1]
```diff
@@ -27,6 +32,14 @@ export function AccountInfoPanel() {
         </div>
       </div>
       <hr className="h-px self-stretch border-transparent bg-zinc-400 bg-opacity-20" />
+      <div className="inline-flex flex-col py-4">
+        <span className="font-inter w-32 text-sm font-medium text-zinc-400">{chain?.name}</span>
+        <span className="font-inter w-32 text-base font-medium text-white">
+          {formatEther(balanceResult.data?.value ?? 0n, 'wei').substring(0, 5)}{' '}
+          {balanceResult.data?.symbol}
```
Comment (Sneh1999) [inline template/web/src/components/header/AccountInfoPanel.tsx:39]: I believe this should be a part of onchain-kit in the avatar component where we can show the user's balance, what are your thoughts on this?

Comment (eragon512) [reply template/web/src/components/header/AccountInfoPanel.tsx:39]: yeah, makes sense to add an onchainkit equivalent here

Comment (Sneh1999) [reply template/web/src/components/header/AccountInfoPanel.tsx:39]: I believe we are building an opinionated codebase, so I believe it should be okay.

Comment (eragon512) [reply template/web/src/components/header/AccountInfoPanel.tsx:39]: PR in the onchainkit repo for this: https://github.com/coinbase/onchainkit/pull/155

File: template/web/src/components/header/AccountInfoPanel.tsx [hunk 2]
```diff
@@ -27,6 +49,14 @@ export function AccountInfoPanel() {
         </div>
       </div>
       <hr className="h-px self-stretch border-transparent bg-zinc-400 bg-opacity-20" />
+      <div className="inline-flex flex-col py-4">
+        <span className="font-inter w-32 text-sm font-medium text-zinc-400">{chain?.name}</span>
+        <span className="font-inter w-32 text-base font-medium text-white">
+          {/* TODO: replace with OnchainKit component */}
+          {parseFloat(formatEther(balance)).toFixed(3)}{' ETH'}
```
Comment (Sneh1999) [inline template/web/src/components/header/AccountInfoPanel.tsx:56]: I think we should merge the onchain kit PR first and then merge this one, what do you think?

Comment (eragon512) [reply template/web/src/components/header/AccountInfoPanel.tsx:56]: cool

**Pull request closed without merging**